### PR TITLE
=doc update stream-io doc page to the latest API

### PR DESCRIPTION
**Pull request by 2m**
Updated Stream Io doc pages to conform to the latest API.

Also fixed style sheets to remove double border from inline code blocks. Inline code now looks like:

selection_007

### CODE REVIEW

**Comment by 2m** [comment]
Failure is unrelated.

File: akka-docs-dev/_sphinx/themes/akka/static/docs.css [hunk 1]
```diff
@@ -170,11 +170,10 @@ strong {color: #0B5567; }
   box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.25);
 }
 
-.pre { padding: 1px 2px; color: #0B5567; background-color: #EFF2F5; border: 1px solid #dee1e2; font-family: Menlo, Monaco, "Courier New", monospace; font-size: 12px; -webkit-border-radius: 3px; -moz-border-radius: 3px; border-radius: 3px; }
+.pre { color: #0B5567; }
```
Comment (ktoso) [inline akka-docs-dev/_sphinx/themes/akka/static/docs.css]: great, thanks! Can you backport this to other branches as well?

**Comment by patriknw** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM

File: akka-docs-dev/rst/java/stream-io.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ Streaming TCP
 
 Accepting connections: Echo Server
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-In order to implement a simple EchoServer we ``bind`` to a given address, which returns a ``Source[IncomingConnection]``,
+In order to implement a simple EchoServer we ``bind`` to a given address, which returns a ``Source<IncomingConnection, Future<ServerBinding>>``,
```
Comment (ktoso) [inline akka-docs-dev/rst/java/stream-io.rst]: nice catch!

**Comment by ktoso** [comment]
LGTM

File: akka-docs-dev/_sphinx/themes/akka/static/docs.css (hunk 1 above)
Comment (2m) [inline akka-docs-dev/_sphinx/themes/akka/static/docs.css]: Done :) #19691

Comment (ktoso) [inline akka-docs-dev/_sphinx/themes/akka/static/docs.css]: Thanks, awesome :)

**Pull request merged**